__tests__/api.test.js

```diff
@@ -76,6 +76,31 @@ describe("/api/articles", () => {
         .expect(400);
       expect(body.message).toBe("Bad Request");
     });
+    it("PATCH: 200 sends the updated article", async () => {
+      const { body } = await request(app).patch("/api/articles/1").send({
+        inc_votes: 10,
+      });
+      expect(body.article[0].votes).toBe(110);
+    });
```

Comment:
I don't think article should be an array - this is on a specific article. You should definitely check that the article_id is what you expect it to be also :)